README.md

```diff
@@ -391,6 +391,35 @@ For more examples and ideas, visit:
 
 ```
 
+#### Proxy Setup
```

Comment:
I think you should add a few lines on why someone would need to set the proxy env variables. e.g.
If the device is located behind a corporate proxy you may need to set the correct environment variables. This is done by configuring proxy behavior for dockerd in the daemon.json file as described in [Configure the Docker daemon to use a proxy server](https://docs.docker.com/config/daemon/systemd/#httphttps-proxy)][https://docs.docker.com/config/daemon/systemd/#httphttps-proxy].

To do this for the application you can either use [developer mode](#developermode) or build the application manually with a pre-creaeted daemon.json that is copied onto the device by the postinstall script 
^ The last section should be extended to be a bit more detailed in both cases

I do not think you should add the shell script - only describe what the daemon.json should contain, where it is located on the device and that the application needs to be restarted once it is updated.

Reply:
Removed the script and the note